src/components/progressbar/Progressbar.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from "classnames";
+
+const styles = {
+    variant1: 'h-1.5 rounded bg-laughingOrange',
+    variant2: 'h-1.5 rounded bg-Malibu',
+}
+
+type ProgressBarProps = {
+    title: string;
+    variant: keyof typeof styles;
+    progress: number;
+    classname?: string;
+}
+
+const ProgressBar: React.FC<ProgressBarProps> = (props) => {
+    const {title, progress, classname,variant} = props; 
+
+    const C = classNames(styles[variant],
```

Comment:
Change it to this `const progressBarClassName = classNames(styles[variant], className);`